src/sms-verification/scripts/directives.js

```diff
@@ -0,0 +1,64 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('user_management.sms-verification');
+
+    module.directive('smsVerify', [
+        '$parse',
+        '$routeParams',
+        'smsVerificationFactory',
+        function ($parse, $routeParams, smsVerificationFactory) {
+            return {
+                restrict: 'A',
+                scope: true,
+                templateUrl: 'templates/user_management/sms-verification/verify_form.html',
+                link: function (scope, element, attrs) {
+
+                    // 2-way bind data attribute to scope.
+                    // scope: { data: '=' }
+                    if (angular.isDefined(attrs.data)) {
```

Comment:
@grahamgilchrist do you think it would be better to have an isolated scope `{ data: '=?' }` or even {` activation_code: '=?',  phone_number: '=?'}`?